plaso/parsers/sqlite_plugins/tango_android.py

```diff
@@ -0,0 +1,332 @@
+# -*- coding:utf-8 -*-
+"""Parser for Tango on Android databases."""
+
+from __future__ import unicode_literals
+
+import codecs
+
+from dfdatetime import java_time as dfdatetime_java_time
+from dfdatetime import semantic_time as dfdatetime_semantic_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class TangoAndroidMessageEventData(events.EventData):
+  """Tango on Android message event data.
+
+  Attributes:
+    message_id (int): message id.
+    payload (str): binary blob with message metadata.
+    direction (int): flag indicating direction of the message.
+  """
+
+  DATA_TYPE = 'tango:android:message'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(TangoAndroidMessageEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.message_id = None
+    self.payload = None
+    self.direction = None
+
+
+class TangoAndroidConversationEventData(events.EventData):
+  """Tango on Android conversation event data.
+
+  Attributes:
+    conversation_id (int): conversation id.
+    payload (str): binary blob with conversation metadata.
+  """
+
+  DATA_TYPE = 'tango:android:conversation'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(TangoAndroidConversationEventData,
+          self).__init__(data_type=self.DATA_TYPE)
+    self.conversation_id = None
+    self.payload = None
+
+
+class TangoAndroidContactEventData(events.EventData):
+  """Tango on Android contact event data.
+
+  Attributes:
+    first_name (str): contact profile first name.
+    last_name (str): contact profila last name.
+    birthday (str): contact profile birthday.
+    gender (str): contact profile gender.
+    status (str): contact status message.
+    distance (int): contact profile distance.
```

Comment:
in what units?

Reply:
it's not clear from the database or documentation the units.